README.md

```diff
@@ -412,6 +419,7 @@ The `{% sub_menu %}` tag is used within menu templates to render additional leve
 - **`allow_repeating_parents`**: By default, the tag will inherit this behaviour from whatever was specified for the original menu tag. However, you can override that behaviour by adding either `allow_repeating_parents=True` or `allow_repeating_parents=False` to the tag in your custom menu template.
 - **`template`** (default: `''`): Lets you render the menu to a template of your choosing. If not specified, the `sub_menu_template` value added to the context by the original menu tag will be used.
 - **`use_specific`**: By default, the tag will inherit this behaviour from whatever was specified for the original menu tag. However, the value can be overridden by passing this option to the {% sub_menu %} tag in your custom menu template. Take a look at the [Specific pages instances and performance](#specific-page-use) section below for a description of the option values supported.
+- **`use_absolute_urls`** (default: `None`): Will use absolute URLs when creating the links for pages. When `None` is specified we check the template context for `use_absolute_urls`, this allows the sub menu to honor the value specified for any parent menus. If `use_absolute_urls` is not in the context and not passed to the tag, it defaults to `False`.
```

Comment:
And could this one be something like:

> By default, the tag will inherit this behaviour from whatever was specified for the original menu tag. However, this can be overridden by explicitly adding `use_absolute_urls=True` or `use_absolute_urls=False` to a `{% sub_menu %}` tag in a custom menu template. If `True`, absolute page URLs will be used in the menu instead a relative URLs.